diff --git a/code/src/rpdhg_clp_gpu/cuda/sufficient_block_proj.cu b/code/src/rpdhg_clp_gpu/cuda/sufficient_block_proj.cu
index 8dcaf91..7a07790 100644
--- a/code/src/rpdhg_clp_gpu/cuda/sufficient_block_proj.cu
+++ b/code/src/rpdhg_clp_gpu/cuda/sufficient_block_proj.cu
@@ -1253,7 +1253,7 @@ sufficient_block_proj(double* arr, double* bl, double* bu, double* D_scaled, dou
     // all threads for positive projection
     long n = ns[0];
     double *sol = arr + gpu_head_start[0];
-    for (int i = global_thread_idx; i < n; i += total_thread){
+    for (int i = global_thread_idx; i < n; i += total_threads){
       sol[i] = fmax(sol[i], 0.0);
     }
   }
@@ -1269,7 +1269,7 @@ sufficient_block_proj(double* arr, double* bl, double* bu, double* D_scaled, dou
     // all threads for positive projection
     long n = ns[2];
     double *sol = arr + gpu_head_start[2];
-    for (int i = global_thread_idx; i < n; i += total_thread){
+    for (int i = global_thread_idx; i < n; i += total_threads){
       sol[i] = fmax(sol[i], 0.0);
     }
   }
@@ -1350,4 +1350,4 @@ sufficient_block_proj(double* arr, double* bl, double* bu, double* D_scaled, dou
       }
     }
   }
-}
\ No newline at end of file
+}
